Framework/MDAlgorithms/src/PreprocessDetectorsToMD.cpp

```diff
@@ -251,33 +253,27 @@ void PreprocessDetectorsToMD::processDetectorsPositions(
     Azimuthal[i] = std::numeric_limits<double>::quiet_NaN();
     //     detMask[i]  = true;
 
-    // get detector or detector group which corresponds to the spectra i
-    Geometry::IDetector_const_sptr spDet;
-    try {
-      spDet = inputWS->getDetector(i);
-    } catch (Kernel::Exception::NotFoundError &) {
-      continue;
-    }
-
     // Check that we aren't dealing with monitor...
```

Comment:
This comment is a bit misleading now, the code is pretty obvious so I think it should just be removed.